src/axolotl/utils/callbacks.py

```diff
@@ -561,9 +563,12 @@ def on_train_begin(
     ):
         if is_main_process():
             try:
-                artifact = wandb.Artifact(name="axolotl-config", type="config")
-                artifact.add_file(local_path=self.axolotl_config_path)
-                wandb.run.log_artifact(artifact)
+                # sync config to top level in run, cannot delete file right away because wandb schedules it to be synced even w/policy = 'now', so let OS delete it later.
+                with NamedTemporaryFile(
+                    mode="w", delete=False, suffix=".yml", prefix="axolotl_config_"
+                ) as temp_file:
+                    copyfile(self.axolotl_config_path, temp_file.name)
+                    wandb.save(temp_file.name)
                 LOG.info("Axolotl config has been saved to WandB as an artifact.")
```

Comment:
I tested this and it works. I think we should update this message to find it in the `Files` section instead.

Reply:
@NanoCode012 sounds good I made the change!